unfoldingWord/translationCore
# Percentage Complete For Word Alignment Tool

#### This pull request addresses:
This PR adds support for finding how much a user has done in the word alignment tool and displaying it in the home page and the tool page.
Percentage based on number of words aligned;

#### How to test this pull request:
Check out the same branch under the tC_resources repo. You will need the index.json file from this PR
https://github.com/translationCoreApps/tC_Resources/pull/10
Open a titus project and use the alignment tool and make a few checks to see the percentage go up.
Make sure you can see it in the home overview and the actual tool card when loading a tool.


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/3013)
